maintenance/fixconfig/main.go

```diff
@@ -121,6 +124,45 @@ func convertJobToSecurityJob(j *config.Presubmit) {
 				},
 			},
 		)
+		// strip cache disk related args etc
+		filteredArgs := []string{}
+		for _, arg := range container.Args {
+			if strings.HasPrefix(arg, "--git-cache") {
+				continue
+			}
+			filteredArgs = append(filteredArgs, arg)
+		}
+		container.Args = filteredArgs
+		// filter cache related env
+		filteredEnv := []kube.EnvVar{}
+		for _, env := range container.Env {
+			// don't keep bazel cache directory env
+			if env.Name == "TEST_TMPDIR" {
+				continue
+			}
+			filteredEnv = append(filteredEnv, env)
+		}
+		container.Env = filteredEnv
+		// filter cache disk volumes
+		filteredVolumes := []kube.Volume{}
+		removedVolumeNames := sets.String{}
+		for _, volume := range j.Spec.Volumes {
+			if volume.VolumeSource.HostPath != nil &&
+				strings.HasPrefix(volume.VolumeSource.HostPath.Path, "/mnt/disks/ssd0") {
+				removedVolumeNames.Insert(volume.Name)
+				continue
+			}
+			filteredVolumes = append(filteredVolumes, volume)
+		}
```

Comment:
assume you need to set `j.Spec.Volumes = filteredVolumes`

Reply:
done, thanks